infra/resources/modules/commons/api/ts_management/v1/_openapi.yaml

```diff
@@ -474,3 +538,19 @@ components:
                   type: string
                   description: The identity ID used to receive events from the queue.
                   minLength: 1
+    TrialForCollection:
```

Comment:
**question(non-blocking)**: what do you think of a name like `TrialSlim` which represents a slim version of the `Trial`?

Anyway, it's up to you decide what naming to use :)

Reply:
Sounds good! Honestly I'm not that good at coming up with new names ahah